ginga/util/plots.py

```diff
@@ -467,12 +467,12 @@ def plot_fwhm(self, x, y, radius, image, cutout_data=None,
 
             self.logger.debug("xarr=%s" % (str(xarr)))
             fwhm_x = self._plot_fwhm_axis(xarr, iqcalc, skybg,
-                                          'blue', 'blue', 'skyblue',
+                                          '#af8dc3', '#af8dc3', 'thistle',
                                           fwhm_method=fwhm_method)
 
             self.logger.debug("yarr=%s" % (str(yarr)))
             fwhm_y = self._plot_fwhm_axis(yarr, iqcalc, skybg,
-                                          'green', 'green', 'seagreen',
+                                          '#7fbf7b', '#7fbf7b', 'aquamarine',
```

Comment:
Try `#1b9e77`

These are also suggested by the color brewer as safe for color blind. I think it is bolder and easier to read. What do you think?

Once we decide on the color, we should also modify [here](https://github.com/ejeschke/ginga/blob/master/ginga/util/plots.py#L400) with the same substitutions for `blue` and `green`. This will fix a colorblind issue for the `Radial` plot as well.